src/components/ListItem/ListItem.tsx

```diff
@@ -30,42 +30,90 @@ export const ListItem: React.FC<ListItemProps> = ({
             tabIndex={tabIndex}
             {...props}
         >
-            {
-                iconStart && !image && (
-                    <div className="moonstone-listItem_iconStart">
-                        <iconStart.type {...iconStart.props} size={iconSize}/>
-                    </div>
-                )
-            }
+            <div className="flexRow flexRow_nowrap alignCenter widthInherit">
 
-            {
-                image && !iconStart && (
-                    <figure className={clsx(
-                        `moonstone-listItem-image_${imageSize}`,
-                        'flexRow',
-                        'alignCenter'
-                    )}
-                    >
-                        {image}
-                    </figure>
-                )
-            }
+                {
+                    iconStart && !image && (
+                        <div className="moonstone-listItem_iconStart">
+                            <iconStart.type {...iconStart.props} size={iconSize}/>
+                        </div>
+                    )
+                }
+
+                {
+                    image && !iconStart && (
+                        <figure className={clsx(
+                            `moonstone-listItem-image_${imageSize}`,
+                            'flexRow',
+                            'alignCenter'
+                        )}
+                        >
+                            {image}
+                        </figure>
+                    )
+                }
+
+                {
+                    image && !iconStart && (
+                        <div className="flexCol">
+                            <Typography
+                                isNowrap
+                                isHtml={isHtml}
+                                className={clsx('flexFluid')}
+                                variant={typographyVariant}
+                                component="span"
+                            >
+                                {label}
```

Comment:
I don't clearly understand the conditions to display the label of the item - it seems that no label is displayed when `image && iconStart` , is it ok ?? It was inconditionnaly displayed before

Reply:
I simplify the component it should be clearer